config/config.exs

```diff
@@ -1,4 +1,16 @@
 
 use Mix.Config
 
-config :cottontail, amqp_url: System.get_env("AMQP_URL")
+url = System.get_env("AMQP_URL")
+
+config :cottontail, amqp_url: url
+config :cottontail, defaults: [
+  queue: [
+    url: url,
+    exchange: {"test.main", :direct}
```

Comment:
Should this also come from env? Or does it get added somewhere else?

Reply:
This is just the default config for the lib itself which isn't going to get loaded by apps which use it, if someone using the app want's to load it from the env then they can just add it to their own app config.